draft-dekater-scion-controlplane.md

```diff
@@ -1461,9 +1472,77 @@ When the segment-request handler of a *core AS* control service receives a path
 
 # Security Considerations
 
-One of the fundamental objectives that guided the design of SCION is security, in particular network security. See chapter 7 of the SCION book (Security Analysis), which states the precise security goals of various network participants and how SCION achieves these goals in the presence of different types of adversaries {{CHUAT22}}.
+This section describes the possible security risks and attacks that SCION's control plane may be prone to, and how these may be mitigated. The focus lies on *inter*-AS routing: SCION does not solve intra-AS routing issues, nor does it provide end-to-end payload encryption, and identity authentication. These topics lie therefore outside the scope of this section.
+
+As described previously, the goal of SCION’s beaconing process is to discover paths between any two ASes and make them available to other ASes. This section examines several possible approaches to manipulating the beaconing process in the SCION control plane, and shows for each case to what extent SCION's design can prevent the corresponding attack or help to mitigate it.
+
+- Path hijacking through interposition (see [](#path-hijack))
+- Creation of spurious ASes (see [](#fake-ases))
+- Peering link misuse (see [](#peer-link-misuse))
+- Manipulation of the path selection process (see [](#manipulate-selection))
+
+**Note:** This section only discusses SCION control plane- and routing-specific security considerations. For security considerations related to the SCION control-plane PKI, see {{I-D.scion-cppki}}. {{I-D.scion-dp}} includes security considerations that concern the SCION data plane and data forwarding.
+
+
+## Path Hijacking through Interposition {#path-hijack}
+
+To attract traffic and to include its own AS in segments and paths, an adversary might try to manipulate the beaconing process. There are several imaginable options for a malicious AS M to do this:
```

Comment:
What's "AS M" in "for a malicious AS M"? Oh, I get it. It's a name for the hypothetical AS that is malicious. The phrasing doesn't make it too obvious.

I would suggest a few things:
* In security circles, the good guys trying to communicate are typically called A (for Alice), and B (for Bob), while the villain is C (for Charlie).
* May be start with. In what follows, we postulate two legitimate ASes, A (operated by Alice) and B (operated by Bob), and a malicious AS C, operated by Charlie.

That might make it easier to describe motivations and methods.

Reply:
@jiceatscion, thank you for your suggestion!
I have removed the M for malicious, and also rephrased the text a bit. Hope it is more clear now.